includes/payment-methods/class-wc-stripe-upe-payment-method-cc.php

```diff
@@ -123,4 +123,13 @@ public function create_payment_token_for_user( $user_id, $payment_method ) {
 		return $token;
 	}
 
+	/**
+	 * Returns boolean dependent on whether capability
+	 * for site account is enabled for payment method.
+	 *
+	 * @return bool
+	 */
+	public function is_capability_active() {
+		return true;
+	}
```

Comment:
Yay or nay?

Reply:
I like this, in the non-UPE version, Card was always on anyway